### PR TITLE
Set correct key for HPC product selection on ARM

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: tests/installation/welcome.pm [hunk 1]
```diff
@@ -128,4 +124,3 @@ sub post_fail_hook {
 }
 
 1;
-# vim: set sw=4 et:
```
Comment (okurz) [inline tests/installation/welcome.pm]: Don't remove this here in a commit that is about something else. Better change it in all files and propose a PR

File: tests/installation/welcome.pm [hunk 2]
```diff
@@ -95,13 +95,9 @@ sub run {
...
-            my %hotkey = (
-                sles     => 's',
-                sled     => 'u',
-                sles4sap => 'i',
-                hpc      => 'x'
-            );
-            $hotkey{sles4sap} = 'u' if get_var('OFW');
+            my %hotkey = (sles => 's', sled => 'u');
+            $hotkey{hpc} = check_var('ARCH', 'x86_64') ? 'x' : 'u';
+            $hotkey{sles4sap} = get_var('OFW') ? 'u' : 'i';
```
Comment (okurz) [inline tests/installation/welcome.pm]: Better assign based on conditions within the hash.

File: tests/installation/welcome.pm [hunk 3]
```diff
@@ -98,10 +98,9 @@ sub run {
             my %hotkey = (
                 sles     => 's',
                 sled     => 'u',
-                sles4sap => 'i',
-                hpc      => 'x'
+                sles4sap => get_var('OFW') ? 'u' : 'i',
```
Comment (asmorodskyi) [inline tests/installation/welcome.pm]: @ldevulder is it ok ? I haven't change a flow . just refactor a bit.

Comment (ldevulder) [reply tests/installation/welcome.pm]: Sounds good to me, but sles4sap tests have been done by @alvarocarvajald, so I let him the final word ;-)

Comment (alvarocarvajald) [reply tests/installation/welcome.pm]: LGTM

**Pull request merged**